### DISCUSSION TITLE
✅ "Select Parent Element" Button

### DISCUSSION THREAD

**Discussion by hamdirizal** [elementor/elementor#13834] [Feature Request]
Hi. I have been using Elementor for a while now. 
On so many occasions when sections are overlapping, sometimes it is hard to select a section and can only select Elements inside that section.

So my idea is, can you add a button to select the parent of the currently selected element.
For example, if we currently selecting a widget, clicking that button will select the column, and clicking it again will select the section.

This feature exists in the other Page Builders, so I think it is a good idea to implement it on Elementor too.

**Comment by drinkmaker** [comment]
And how about using Navigator? https://elementor.com/help/navigator/

**Reply by hamdirizal** [reply]
Oh, man. 

I never knew about that feature. It is there the whole time. 

Alright, please ignore my idea. The current navigator is cool and more than enough.

Thanks.

**Comment by nicholaszein** [comment]
Hi, as @hamdirizal, mentioned, this feature was added in August, 2018, with the release of version 2.2.0.

Just use the Navigator/Structure Panel: https://elementor.com/help/navigator/

Kind regards
